contrib/python-pytest-mock/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "python-pytest-mock"
+pkgver = "3.12.0"
+pkgrel = 0
+build_style = "python_pep517"
+
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+checkdepends = ["python-pytest-asyncio"]
+depends = ["python-mock", "python-pytest"]
+pkgdesc = "Thin-wrapper around the mock package for easier use with pytest"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MIT"
+url = "https://pytest-mock.readthedocs.io/en/latest/index.html"
+source = f"https://github.com/pytest-dev/pytest-mock/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "aeedddfe2feefde936ba3d723e26db5757570537ab66bf4dd2cd17d4c263f022"
+
+
+def init_build(self):
+    self.env["SETUPTOOLS_SCM_PRETEND_VERSION"] = f"{pkgver}"
+
+
+def init_check(self):
+    self.make_check_args = [
+        "--deselect",
+        "tests/test_pytest_mock.py::TestMockerStub::test_failure_message_with_",
+    ]
```

Comment:
make_check_args toplevel

Reply:
also why does that fail?

Reply:
this is the error https://github.com/chimera-linux/cports/actions/runs/7601459852/job/20700867634#step:8:725

Reply:
https://github.com/pytest-dev/pytest-mock/pull/403

Reply:
yes thats it, guess its not worth adding 3 commits to fix the test

Reply:
it's one commit https://github.com/pytest-dev/pytest-mock/commit/3d48ff90257daf6b3c50100bd5dd36faf34e70dc

Reply:
you can use https://github.com/pytest-dev/pytest-mock/commit/6da5b0506d6378a8dbe5ae314d5134e6868aeabd.patch to shrink the patch to 1 commit too

Reply:
okay